html/captive-portal/lib/captiveportal/PacketFence/Controller/Violation.pm

```diff
@@ -92,6 +94,29 @@ sub index : Path : Args(0) {
     }
 }
 
+=head2 getSubTemplate
+
+Get the subtemplate in the right portal profile
+
+=cut
+
+sub getSubTemplate {
+    my ( $self, $c, $template ) = @_;
+    my $portalSession = $c->portalSession;
+    return "violations/$template.html";
+#    my $langs         = $portalSession->getRequestLanguages();
+    my $langs         = [];
```

Comment:
So looking at it, seems we do support them (well we ship a few of the templates with locales in them)